src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -20,7 +20,7 @@
         "ext-xml": "*",
         "symfony/config": "^4.2|^5.0",
         "symfony/dependency-injection": "^4.2|^5.0",
-        "symfony/http-kernel": "^4.3|^5.0",
+        "symfony/http-kernel": "^4.3",
```

Comment:
because of https://github.com/symfony/symfony/blob/4.4/src/Symfony/Bundle/SecurityBundle/EventListener/FirewallListener.php#L58